tembo-cli/src/cmd/apply.rs

```diff
@@ -748,31 +754,36 @@ pub fn get_rendered_dockerfile(
     Ok(rendered_dockerfile)
 }
 
-fn get_postgres_config(instance_settings: &InstanceSettings) -> String {
+fn get_postgres_config(
+    instance_ps_config: Option<HashMap<String, Value>>,
+    postgres_configs: &std::option::Option<Vec<ControllerPgConfig>>,
+) -> String {
     let mut postgres_config = String::from("");
     let qoute_new_line = "\'\n";
     let equal_to_qoute = " = \'";
-    let stack_type = ControllerStackType::from_str(instance_settings.stack_type.as_str())
-        .unwrap_or(ControllerStackType::Standard);
-
-    let stack = get_stack(stack_type);
-
-    if stack.postgres_config.is_some() {
-        for config in stack.postgres_config.unwrap().iter() {
-            postgres_config.push_str(config.name.as_str());
-            postgres_config.push_str(equal_to_qoute);
-            postgres_config.push_str(format!("{}", &config.value).as_str());
-            postgres_config.push_str(qoute_new_line);
+    if postgres_configs.is_some() {
+        for p_config in postgres_configs.clone().unwrap().into_iter() {
+            match p_config.value {
+                ControllerConfigValue::Single(val) => {
```

Comment:
Another way you could do this is with the `write!` macro, e.g.

```suggestion
      writeln!(postgres_config, "{} = '{val}'", p_config.name.as_str())?;
```

Behaves just like the usual `println!` macro

Reply:
Much cleaner. Changed it everywhere